modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -227,18 +227,22 @@ export default class Tile3DLayer extends CompositeLayer {
 
   _makeSimpleMeshLayer(tileHeader, oldLayer) {
     const content = tileHeader.content;
-    const {attributes, modelMatrix, cartographicOrigin, texture} = content;
-    const {positions, normals, texCoords} = attributes;
+    const {modelMatrix, cartographicOrigin, texture} = content;
+    const attributes = {};
+    attributes.positions = content.attributes.positions;
+    attributes.positions.value = new Float32Array(attributes.positions.value);
+    if (content.attributes.normals) {
+      attributes.normals = content.attributes.normals;
+    }
+    if (content.attributes.texCoords) {
+      attributes.texCoords = content.attributes.texCoords;
+    }
```

Comment:
I am not sure why we are converting positions to `Float32Array` here in the first place. I actually recommend we instead remove that and just pass on positions.

The accessor can have type information that is different from float32.

Reply:
There is `Float64Array`. It is not supported by following code in deck.gl/luma.gl (error occurs)

Reply:
Hmm, we do have advanced processing for Float64 arrays in deck and luma.

Does I3S use Float64 for positions?

Reply:
It transforms from Float32 to Float64:
https://github.com/visgl/loaders.gl/blob/f48ce79162d802f9c7ef59f6030821a15bcba5b1/modules/i3s/src/lib/parsers/parse-i3s-tile-content.js#L283

I suppose it is done to save precision.

Reply:
> we do have advanced processing for Float64 arrays in deck and luma

64 bit position is supported in deck attributes, not luma. The mesh is directly passed to luma's Model.

Reply:
Yes good point. Though I am not quite sure this is the best place to do the conversion. Seems like either in a higher layer or in the model itself...

Reply:
Best if this layer just takes the same attributes as a luma model, otherwise we need to add more documentation about this layer.

Reply:
Again: I just keep existing logic here. The PR is about making `normals` optional. I suggest merge this PR if no more comments. I really need this changes to keep going with DRACO support in I3S.